hdtv/rootext/mfile-root/mfile/src/trixi_minfo.c

```diff
@@ -73,34 +74,35 @@ void trixi_probe(MFILE *mat) {
   trixi_header trixi_h;
   char comment[512];
 
-  if (_get (mat->ap, (char *)&trixi_h, 0, sizeof(trixi_h)) != sizeof(trixi_h))
+  if (_get(mat->ap, (char *)&trixi_h, 0, sizeof(trixi_h)) != sizeof(trixi_h))
     return;
 
-  if (strncmp (trixi_h.code, trixi_magic, strlen(trixi_magic)) != 0) return;
+  if (strncmp(trixi_h.code, trixi_magic, strlen(trixi_magic)) != 0)
```

Comment:
Codacy found an issue: [Does not handle strings that are not \0-terminated; if given one it may perform an over-read (it could cause a crash if unprotected) (CWE-126).  ](https://app.codacy.com/manual/janmayer/hdtv/pullRequest?prid=6293112)